documentation/modules/rn-2.4.adoc

```diff
@@ -163,3 +159,6 @@ The Universal Unique ID (UUID) number within the System Management BIOS (SMBIOS)
 
 Previously, the password that was specified for {rhv-short} manager appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for {rhv-short} manager.
 
+.QEMU guest agent is not installed on migrated VMs
+
+The QEMU guest agent is not installed on migrated VMs. Workaround: Install the QEMU guest agent with a post-migration hook. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[(BZ#2018062)]
```

Comment:
this should rather now be something like:
```suggestion
The QEMU guest agent is installed on VMs during cold migration from vSphere. link:https://issues.redhat.com/browse/MTV-332[(MTV-332)]
```

Reply:
Fixed -- also changed "not" in header to "now".

Reply:
@anarnold97 2nd eyes on this? Would you go with "not installed," "now installed," or simply "installed" ?